src/servers/src/auth.rs

```diff
@@ -108,6 +108,12 @@ pub enum Error {
     #[snafu(display("Encounter IO error, source: {}", source))]
     IOErr { source: std::io::Error },
 
+    #[snafu(display("Encounter system error, msg: {}", msg))]
+    SystemErr { msg: String },
+
+    #[snafu(display("Rpc call failed, {}: {}", code, msg))]
+    RpcResultErr { code: u32, msg: String },
```

Comment:
These two error is not meaningful, too broad, better consider a more detailed name.